src/Utils/Html.php

```diff
@@ -363,17 +363,39 @@ public function getText()
 
 
 	/**
+	 * @deprecated
+	 */
+	public function add($child)
```

Comment:
I like how is the add() short. What about to allow HTML instance and strings consider as a text? Not sure how to warn about change.